src/rgw/rgw_crypt.cc

```diff
@@ -24,6 +24,24 @@
 #include <cryptopp/cryptlib.h>
 #include <cryptopp/modes.h>
 #include <cryptopp/aes.h>
+#if defined(CIVETWEB_VERSION_MAJOR) && defined(CIVETWEB_VERSION_MINOR) \
+  && ((CIVETWEB_VERSION_MAJOR>=1 && CIVETWEB_VERSION_MINOR>=10) || \
+      (CIVETWEB_VERSION_MAJOR>1))
```

Comment:
i think this line should be 
```c
   (CIVETWEB_VERSION_MAJOR>1))
```

but maybe following macro is easier to digest?
```c
#if defined(CIVETWEB_VERSION_MAJOR) && \
    defined(CIVETWEB_VERSION_MINOR) &&
    (CIVETWEB_VERSION_MAJOR * 100) + CIVETWEB_VERSION_MINOR >= 110
```

Reply:
@tchaikov Basically I like your solution better.